logger.go

```diff
@@ -74,16 +72,41 @@ func getLogLevel(logLevel string) zapcore.Level {
 	}
 }
 
+type Error interface {
+	Message() string
+	Error() string
+}
+
+type helperError struct {
+	message  string
+	rawError string
+}
+
+func (e helperError) Message() string {
+	return e.message
+}
+
+func (e helperError) Error() string {
+	return e.rawError
```

Comment:
I suggest you return the message here. Make a RawError function to return the raw error. What say?

Reply:
Yes